include/ignition/utils/config.hh

```diff
@@ -15,4 +15,14 @@
  *
  */
 
+#ifndef IGNITION_UTILS__CONFIG_HH_
+#define IGNITION_UTILS__CONFIG_HH_
+
 #include <gz/utils/config.hh>
+
+namespace [[deprecated("Use gz namespace.")]]  ignition
```

Comment:
can we add the namespace for now without the deprecation? merging this will introduce compiler warnings in all downstream packages. we can add the deprecation after we've already converted everything to `gz`

Reply:
Now we're doing pragma message deprecation! https://github.com/gazebosim/gz-utils/pull/61/commits/5e26778ffd5aef93ac7f4b031c0b30cd57f4e600